sdks/python/container/Dockerfile

```diff
@@ -43,9 +45,13 @@ RUN \
     # Remove pip cache.
     rm -rf /root/.cache/pip
 
+# Configure ccache prior to installing the SDK.
+RUN ln -s /usr/bin/ccache /usr/local/bin/gcc
+# These parameters are needed as pip compiles artifacts in random temporary directories.
+RUN ccache --set-config=sloppiness=file_macro && ccache --set-config=hash_dir=false
 
 COPY target/apache-beam.tar.gz /opt/apache/beam/tars/
-RUN pip install /opt/apache/beam/tars/apache-beam.tar.gz[gcp] && \
+RUN pip install -v /opt/apache/beam/tars/apache-beam.tar.gz[gcp] && \
     # Remove pip cache.
     rm -rf /root/.cache/pip
 
```

Comment:
Could we add a `pip freeze --all`? From the logs that will give us a complete list of what exact versions are installed.

Reply:
Done.